internal/weldr/api.go

```diff
@@ -26,27 +28,16 @@ type API struct {
 	router *httprouter.Router
 }
 
-func New(repo rpmmd.RepoConfig, packages rpmmd.PackageList, logger *log.Logger, initialState []byte, stateChannel chan<- []byte, jobs chan<- job.Job, jobStatus <-chan job.Status) *API {
+func New(repo rpmmd.RepoConfig, packages rpmmd.PackageList, logger *log.Logger, store *store.Store) *API {
 	// This needs to be shared with the worker API so that they can communicate with each other
 	// builds := make(chan queue.Build, 200)
 	api := &API{
-		store:    newStore(initialState, stateChannel, jobs, jobStatus),
+		store:    store,
 		repo:     repo,
 		packages: packages,
 		logger:   logger,
 	}
 
-	// sample blueprint on first run
-	if initialState == nil {
-		api.store.pushBlueprint(blueprint.Blueprint{
-			Name:        "example",
-			Description: "An Example",
-			Version:     "1",
-			Packages:    []blueprint.Package{{"httpd", "2.*"}},
-			Modules:     []blueprint.Package{},
-		})
-	}
```

Comment:
I'm fine with removing this, because it's a silly example. However, Image Builder always had a few examples in it. I included this here originally to verify that this approach would work.